### PR TITLE
Normalize data 

### PR DESCRIPTION
**Pull request by JuliaLinhart**
Normalize `theta_est` in the same way as `theta_ref` before computing metrics like `c2st`.

### CODE REVIEW

File: objective.py [hunk 1]
```diff
@@ -60,21 +111,57 @@ def compute(
...
             n_ref = (theta.shape[0] for theta in self.theta_ref)
+            # same conditioning observation as for the reference posterior
             theta_est = [sample(x, n) for x, n in zip(self.x_ref, n_ref)]
             end = time.perf_counter()
 
+            # Normalize samples.
+            theta_est = [
+                (theta - self.mean_theta) / self.std_theta
+                for theta in theta_est
+            ]
```
Comment (francois-rozet) [inline objective.py]: This is incorrect. The estimators have been trained on normalized data, so they should generate normalized data already.

However we probably have an issue with the sampling from NRE and sbi's NPE, which uses a prior that does not match the normalized parameters. We can solve this by storing a transformed prior with respect to `theta_mean` and `theta_std`. I can do it if necessary.

**Review by francois-rozet — CHANGES_REQUESTED** [review]

File: objective.py (hunk 1 above)
Comment (JuliaLinhart) [reply objective.py]: Oh okay yes indeed my tests fail.. And yes please! 
Do we keep the z-scoring in the c2st metric?

Comment (francois-rozet) [reply objective.py]: I submitted a PR (#12) to fix the issue. I did some tests, and it seems to work on my machine. For the C2ST, it is not necessary to re-scale the data if they are already standardized.

**Comment by francois-rozet** [comment]
Shouldn't we close this branch?

**Comment by JuliaLinhart** [comment]
Yes absolutely ! I just didn’t want to do it without your appoval

**Comment by JuliaLinhart** [comment]
I'm closing this PR, with the only change being that by default `z_score=False` in `c2st`!

**Pull request merged**